dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,67 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseOption):
+    def __init__(self):
+        """
+        Constructs the TextProfilerOption object with default values.
+
+        :ivar is_case_sensitive: option set for case sensitivity.
+        :vartype is_case_sensitive: bool
+        :ivar stop_words: option set for stop words.
+        :vartype stop_words: Union[None, list(str)]
+        :ivar words: option set for word update.
+        :vartype words: BooleanOption
+        :ivar vocab: option set for vocab update.
+        :vartype vocab: BooleanOption
+        """
+        self.is_case_sensitive = True
+        self.stop_words = None
```

Comment:
Do these add to our existing stop words or overwrite them (if the user specifies them via options)